Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
Don't think the information is actually _missing_.
Are there cases where the FieldDescription is constructed outside `ModelManager::getNewFieldDescriptionInstance()` ? I couldn't find any.
Isn't it more that the builder adds _additional_ information, mainly for building the view ?